ETWAnalyzer/EventDump/DumpFile.cs

```diff
@@ -103,7 +103,8 @@ public override List<MatchData> ExecuteInternal()
                                   "SetSecurity Count",
                                   "SetSecurity Times",
                                   "File Delete Count",
-                                  "File Rename Count"
+                                  "File Rename Count",
+                                  "File Open Close Time"
```

Comment:
Perhaps "File Open/Close Duration (s)"? What are the units? Seconds ms us?

Reply:
resolved